build-system/runner/amp-test/0.1/test.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * Copyright 2016 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+import {dev} from './test-2';
+
+function main() {
+  let b = dev.assert('assert');
+  dev.fine('fine');
+  dev.warn('warn');
+  var a = 1 + 1;
+  return a;
+}
+
+function sideEffectAssert() {
```

Comment:
If that is not what you currently do it may be fine for now, but we should be correct on terminology.

For side effects I'd expects something like

`dev.assert(document.body.appendChild(…))`

to compile to

`document.body.appendChild(…)`